codalab/lib/bundle_cli.py

```diff
@@ -1120,6 +1120,13 @@ def do_workers_command(self, args):
                 help='Upload to this worksheet (%s).' % WORKSHEET_SPEC_FORMAT,
                 completer=WorksheetsCompleter,
             ),
+            Commands.Argument(
```

Comment:
ignore-gitignore is counterintuitive...I'd just make this `--ignore` with the default being '.gitignore'.  People can override this by passing in `''`.

Reply:
@percyliang: Just having it a `--ignore` argument is cleaner to implement, but this would allow the user to specify any ignore file. I'm not sure if that flexibility is needed, since most users will just rely on `.gitignore` and most likely won't add another ignore file like `.clignore` to their repos. And, if that's the case, I think it's cleaner to just rely on a boolean flag to turn off the functionality instead of passing in an empty string.

Reply:
I'm not sure about that...people might have different ignore files for different purposes (e.g., `.dockerignore`).  As I mentioned before, you might want to upload files that are not checked into git (like datasets).

Reply:
@percyliang: That makes sense. I have updated the PR.